python/mach/mach/main.py

```diff
@@ -11,6 +11,9 @@
 import codecs
 import errno
 import importlib
+from sys import version_info
+if version_info.major == 3 and version_info.minor > 9 :
+    import importlib.util
```

Comment:
Based on https://discuss.python.org/t/python3-11-importlib-no-longer-exposes-util/25641/9 it seems that the correct thing to do here is to unconditionally import `importlib.util`. That should make this fix a bit simpler.

Reply:
yes, you are right, just import  importlib.util can fix this issue,